packages/server/lib/gui/events.coffee

```diff
@@ -21,12 +21,23 @@ chromePolicyCheck = require("../util/chrome_policy_check")
 browsers    = require("../browsers")
 konfig      = require("../konfig")
 
+removeUnserializableValues = (obj) =>
+  ## remove values that cannot be cloned
+  ## https://github.com/cypress-io/cypress/issues/6750
+  _.cloneDeepWith obj, (val) =>
+    if _.isFunction(val)
```

Comment:
does this need a check against Element?

Reply:
it's in the `server`, there can't be an `Element`